routes/products.js

```diff
@@ -0,0 +1,39 @@
+//jshint esversion:6
+
+const express = require('express');
+const router = express.Router();
+const Products = require('../db/Products');
+const products = new Products();
+
+router.get('/', (req, res) => {
+  res.send('smoke test');
+});
+
+router.post('/new', (req, res) => {
+  res.send(products.createProduct(req.body.name, req.body.price));
```

Comment:
so this route is calling createProduct twice. It should only call it once at most.